roles/openshift_node/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+# tasks file for openshift_node
+- name: Install OpenShift
+  yum: pkg=openshift-node state=installed
+
+  # fixme: Once openshift stops resolving hostnames for node queries remove this...
+- name: Set hostname to IP Addr (WORKAROUND)
+  command: /usr/bin/hostname {{ oo_bind_ip }}
```

Comment:
@jwhonce as said twiest, it is using the hostname module which is idempotent, i.e.: it won't change the hostname again if it is to its correct value.